lara-typescript/src/section-authoring/api/mock-api-provider.ts

```diff
@@ -174,12 +174,14 @@ const copyPage = (args: {pageId: PageId, destIndex: number}) => {
   return Promise.reject("no source page in copy");
 };
 
-export const updatePage = (id: PageId, changes: Partial<IPage>) => {
-  const indx = pages.findIndex(p => p.id === id);
+export const updatePage = (args: {pageId: PageId, changes: Partial<IPage>}) => {
+  const {pageId, changes} = args;
+  const indx = pages.findIndex(p => p.id === pageId);
   if (indx > -1) {
     const nextPage = {... pages[indx], ...changes };
     pages[indx] = nextPage;
     setSectionPositions(nextPage);
+    Object.assign(nextPage, changes);
```

Comment:
This seems redundant given that `changes` were merged into `nextPage` in line 181.